arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -3751,6 +3951,38 @@
       },
       "description": "The content hash."
     },
+    "RetryHistory": {
+      "type": "object",
+      "properties": {
+        "startTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets the start time."
```

Comment:
1105623: Fix the description for startTime and endTime for all API's to be more descriptive